Stripe/StripeiOS/Resources/StripeiOS.xcassets/Cards/Contents.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Wasn't sure how to exactly test this, so i wrote a script to verify the json contents were correct, then changed `file` to `open` and manually verified each image:

```
for i in `find .| grep "\.imageset$"`; do
    for png in `cat $i/Contents.json  |grep ".png" |awk '{print $3}' |tr -d "\"" | tr -d ","`; do
	file $i/$png
	if [ $? -ne 0 ]; then
	    exit 1
	fi
    done
done
```

Reply:
Xcode will enforce this for us, fortunately!